smelter-core/src/stats/mod.rs

```diff
@@ -60,18 +60,24 @@ impl StatsMonitor {
 }
 
 impl StatsSender {
-    pub fn send(&self, event: StatsEvent) {
-        if let Err(TrySendError::Full(event)) = self.0.try_send(event) {
-            warn!(?event, "Stats channel full")
-        };
+    pub fn send(&self, events: Vec<StatsEvent>) {
```

Comment:
I don't see the logic. Why method that sends a vector of events is named send, and single event send_evetnt. It is inconsitent.

Also consider writing generic version that is just one method that can take either
```
pub fn send<T: IntoIter<StatsEvent>>(&self, events: T) {

}

impl IntoIter for StatsEvent {
...
}
```